programs/programs.json

```
@@ -350,5 +350,17 @@
     "categories": [
       "NFT"
     ]
+  },
+    {
+    "name": "soundwork-contracts",
+    "description": "A decentralized Sound Network that seeks to revolutionize music production and redefine the music industry’s supply chain.",
+    "icon": "https://avatars.githubusercontent.com/u/132207133",
+    "repo": "https://github.com/SoundWorkLabs/marketplace-contracts",
```

Due to the current implementation details of viewing projects from GitHub, linking the program directory URL as the repository URL makes it faster to load the program e.g. https://beta.solpg.io/https://github.com/SoundWorkLabs/marketplace-contracts/tree/main/programs

```suggestion
    "repo": "https://github.com/SoundWorkLabs/marketplace-contracts/tree/main/programs",
```